orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModel.java

```diff
@@ -0,0 +1,135 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.core.AiModel;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleConfig;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.With;
+
+/** Large language models available in Orchestration. */
+@With
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
```suggestion
@With
@EqualsAndHashCode
@ToString
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

.. or make this a record, then you can also drop the implementation for the getters..